master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -956,13 +998,25 @@ private[celeborn] class Master(
     sb.toString()
   }
 
-  override def listPartitionLocationInfo: String = throw new UnsupportedOperationException()
-
-  override def getUnavailablePeers: String = throw new UnsupportedOperationException()
-
-  override def isShutdown: String = throw new UnsupportedOperationException()
-
-  override def isRegistered: String = throw new UnsupportedOperationException()
+  override def exclude(addWorkers: String, removeWorkers: String): String = {
+    val sb = new StringBuilder
+    sb.append("============================ Add/Remove Excluded Workers  Manually =============================\n")
+    val workersToAdd = addWorkers.split(",").filter(_.nonEmpty)
+    val workersToRemove = removeWorkers.split(",").filter(_.nonEmpty)
+    self.send(WorkerExclude(
```

Comment:
You should call `self.askSync` instead of `self.send` here because you have written the response. A user will need to know whether the exclude list is effective or not.